README.md

```diff
@@ -106,40 +106,41 @@ Benchmarks measure memoized value retrieval time using
 [`benchmark-ips`](https://github.com/evanphx/benchmark-ips). Benchmarks are
 run in GitHub Actions and updated in every PR that changes code.
 
-**Values >1.00x represent how much _slower_ each gem’s memoized value retrieval
-is than the baseline.**
+All benchmarks are relative to MemoWise.
+
+**Values >1.00x represent how much _slower_ each gem’s memoized value retrieval is than MemoWise.**
```

Comment:
Here's one potential way to condense this:

```suggestion
## Benchmarks

Benchmarks are run in GitHub Actions, and the tables below are updated with every code change. **Values >1.00x represent how much _slower_ each gem’s memoized value retrieval is than the latest commit of `MemoWise`**, according to [`benchmark-ips`](https://github.com/evanphx/benchmark-ips) (2.9.1).

Results using Ruby 3.0.2:

|Method arguments|`Memery` (1.4.0)|`Dry::Core` (0.7.1)|
|--|--|--|
|`()` (none)|1.14x|16.81x|
|`(a)`|2.02x|9.72x|
|`(a, b)`|1.07x|4.90x|
|`(a:)`|1.79x|18.74x|
|`(a:, b:)`|1.03x|9.94x|
|`(a, b:)`|1.03x|9.88x|
|`(a, *args)`|1.99x|4.63x|
|`(a:, **kwargs)`|1.54x|5.97x|
|`(a, *args, b:, **kwargs)`|0.99x|2.81x|

Results using Ruby 2.7.4 (because these gems raise errors in Ruby 3.x):

|Method arguments|`DDMemoize` (1.0.0)|`Memoizer` (1.0.3)|`Memoized` (1.0.2)|`Memoist` (0.16.2)|
|--|--|--|--|--|
|`()` (none)|26.75x|2.62x|1.20x|3.39x|
|`(a)`|19.58x|14.27x|10.51x|11.83x|
|`(a, b)`|4.14x|2.88x|2.36x|2.68x|
|`(a:)`|27.76x|23.54x|19.87x|21.36x|
|`(a:, b:)`|6.57x|5.57x|4.66x|5.06x|
|`(a, b:)`|6.57x|5.01x|4.38x|4.82x|
|`(a, *args)`|5.38x|4.02x|3.40x|3.48x|
|`(a:, **kwargs)`|3.95x|3.26x|2.92x|3.11x|
|`(a, *args, b:, **kwargs)`|2.77x|2.43x|2.23x|2.25x|
```